ios/utils/UINavigationBar+RNSUtility.mm

```diff
@@ -41,4 +41,44 @@ - (nullable UIView *)rnscreens_findBackButtonWrapperView
   return nil;
 }
 
+- (NSDirectionalEdgeInsets)rnscreens_sumTitleControlDirectionalMargins
+{
+  NSDirectionalEdgeInsets totalMargins = NSDirectionalEdgeInsetsMake(0, 0, 0, 0);
+  UIView *titleControl = RNScreensFindTitleControlInView(self);
+
+  if (!titleControl) {
+    NSLog(@"[RNScreens] _UINavigationBarTitleControl not found under the UINavigationBar hierarchy");
+    return totalMargins;
+  }
+
+  UIView *currentView = titleControl.superview;
```

Comment:
On the snippet you provided in PR description I can observe, that `titleControl` has non-zero insets. Why do we skip them by starting from superview?

Reply:
I'm following the same approach as for the `< iOS26` implementation for which we stopped at `_UINavigationBarTitleControl` level